### PR TITLE
Searchbox. Show error when user enters nis1 address

### PR DESCRIPTION
**Pull request by OlegMakarenko**
image

### CODE REVIEW

File: src/config/default.json [hunk 1]
```diff
@@ -10,6 +10,13 @@
 			"http://api-01.us-west-1.testnet.symboldev.network:3000"
 		]
 	},
+	"nis1Nodes": [
```
Comment (AnthonyLaw) [inline src/config/default.json:13]: Any reason we need this?

File: src/config/i18n/en-us.json [hunk 2]
```diff
@@ -1,5 +1,7 @@
 {
     "langName": "en-us",
+    "errorNisAddressNotAllowed": "You are trying to search the NEM NIS1 address in the Symbol Explorer. Please use NEM Explorer instead",
+    "errorNothingFound": "Nothing found..",
```
Comment (segfaultxavi) [inline src/config/i18n/en-us.json:4]:
```suggestion
    "errorNisAddressNotAllowed": "You entered a NEM NIS1 address in the Symbol Explorer. Please use NEM Explorer instead.",
    "errorNothingFound": "Nothing found.",
```
Could the link to the NEM Explorer be given?

**Review by segfaultxavi — CHANGES_REQUESTED** [review]

File: src/config/default.json (hunk 1 above)
Comment (OlegMakarenko) [reply src/config/default.json]: Changed solution a bit. Node list removed

File: src/config/i18n/en-us.json (hunk 2 above)
Comment (OlegMakarenko) [reply src/config/i18n/en-us.json:4]: Thank you, translation improved.

> Could the link to the NEM Explorer be given?
This is a good idea but we don't have an official explorer for NIS1

**Review by AnthonyLaw — APPROVED** [review]

**Pull request merged**